src/adres-crab/adres-crab.ts

```diff
@@ -59,7 +59,7 @@ export class AdresCrab {
       .ensure('gemeente').required()
       .ensure('postcode').required()
       .ensure('straat').required()
-      .ensure('adres').satisfiesRule('requiredHuisnummer')
+      .ensure('adres').satisfiesRule('requiredHuisnummer').when(() => this.config.huisnummer.required)
```

Comment:
misschien een fallback in steken voor projecten die deze config niet hebben?

Reply:
Die default config met huisnummer bestaat hier 
https://github.com/OnroerendErfgoed/oerelia/blob/7e8a9b4d733c1b0011b4047f2969313feb91d355/src/adres-crab/adres-crab.ts#L16